static/app/utils/analytics/integrations/index.ts

```diff
@@ -125,29 +107,12 @@ export const integrationEventMap: Record<IntegrationAnalyticsKey, string> = {
   'integrations.index_viewed': 'Integrations: Index Page Viewed',
   'integrations.directory_item_searched': 'Integrations: Directory Item Searched',
   'integrations.directory_category_selected': 'Integrations: Directory Category Selected',
-  'integrations.stacktrace_link_cta_dismissed':
-    'Integrations: Stacktrace Link CTA Dismissed',
-  'integrations.stacktrace_start_setup': 'Integrations: Stacktrace Start Setup',
-  'integrations.stacktrace_submit_config': 'Integrations: Stacktrace Submit Config',
-  'integrations.stacktrace_complete_setup': 'Integrations: Stacktrace Complete Setup',
-  'integrations.stacktrace_manual_option_clicked':
-    'Integrations: Stacktrace Manual Option Clicked',
-  'integrations.stacktrace_link_clicked': 'Integrations: Stacktrace Link Clicked',
-  'integrations.reconfigure_stacktrace_setup':
-    'Integrations: Reconfigure Stacktrace Setup',
-  'integrations.stacktrace_docs_clicked': 'Integrations: Stacktrace Docs Clicked',
-
   'integrations.serverless_functions_viewed': 'Integrations: Serverless Functions Viewed',
   'integrations.installation_input_value_changed':
     'Integrations: Installation Input Value Changed',
   'integrations.serverless_function_action': 'Integrations: Serverless Function Action',
   'integrations.cloudformation_link_clicked': 'Integrations: CloudFormation Link Clicked',
   'integrations.switch_manual_sdk_setup': 'Integrations: Switch Manual SDK Setup',
-  'integrations.code_owners_cta_setup_clicked':
-    'Integrations: Code Owners CTA Setup Clicked',
-  'integrations.code_owners_cta_docs_clicked':
-    'Integrations: Code Owners CTA Setup Clicked',
```

Comment:
To prove my point about how these can be missed, take a look at how these two events mistakenly have the same event name, likely because they were copy pasted while being created. It's really easy to miss this stuff!

Reply:
Enums might not have prevented this exact error from happening but, if someone were to copy `integrations.code_owners_cta_docs_clicked` from here because they read "CTA Setup Clicked", they could record the wrong analytics interaction in the product. Having to access CodeownersEvents.DOCS_CTA is a little bit more explicit, and if we want to change the analytics key, its a single line change.